docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -761,6 +761,15 @@ paths:
             description: The new moderation status for the annotation.
             type: string
             enum: [APPROVED, PENDING, DENIED, SPAM]
+        - name: previous_moderation_status
+          in: query
```

Comment:
I copy-pasted this from the other params defined here, but I think this is wrong, and these params should be sent in the body.

Perhaps it works both as body and query params and that's why it was documented like this, but we are definitely sending these ase body patams in the actual implementation 😅 

I'll probably change this separately, as for a PATCH request I think it makes more sense to document these as body params, even if it happens to works when sent as query params.